### DISCUSSION TITLE
Comments: Gmail Smart Compose in Keras and Tensorflow.js

### DISCUSSION THREAD

**Discussion by jiayihu** [jiayihu/blog#65] [Announcements]
This issue is reserved for comments to [Gmail Smart Compose in Keras and Tensorflow.js](https://blog.jiayihu.net/gmail-smart-compose-in-keras-and-tensorflow-js/). Leave a comment below and it will be shown in the blog page.

**Comment by federico-terzi** [comment]
A really interesting and well-written article @jiayihu, it was a very nice read!

PS. I was also very surprised when I saw you are based in Italy :)

Keep up the great work

**Comment by jiayihu** [comment]
Thanks, you have awesome projects too and we're both studying at the master degree :)

**Comment by federico-terzi** [comment]
@jiayihu Thank you :)

**Comment by StarWang** [comment]
Hi @jiayihu , Thanks for the great post.

I have a question related to this: do you use the same RNN cell for both encoder and decoder or do you use different cells that their weights are not shared? I tried to use the same cell in TF Keras and it threw errors when I tried to use the same cell for both encoder and decoder :(
